vulture/utils.py

```diff
@@ -56,6 +56,18 @@ def format_path(path):
     return relpath if not relpath.startswith('..') else path
 
 
+def get_decorator_name(decorator):
+    name = ''
```

Comment:
Strings should be concatenated with `str.join`. Use `parts = []` to collect the parts and then `return '.'.join(parts)`.